mutiny-core/src/lib.rs

```diff
@@ -308,6 +311,58 @@ impl<S: MutinyStorage> MutinyWallet<S> {
         }
     }
 
+    /// Get contacts from the given npub and sync them to the wallet
+    pub async fn sync_nostr_contacts(
+        &self,
+        npub: XOnlyPublicKey,
+        timeout: Option<Duration>,
+    ) -> Result<(), MutinyError> {
+        let keys = Keys::from_public_key(npub);
+        let client = Client::new(&keys);
+
+        #[cfg(target_arch = "wasm32")]
+        client.add_relay("wss://relay.damus.io").await.unwrap();
+
+        #[cfg(not(target_arch = "wasm32"))]
+        client
+            .add_relay("wss://relay.damus.io", None)
+            .await
+            .unwrap();
```

Comment:
If we're doing more nostr things, we should probably make relay(s) configurable. Can be done in a follow up later this month if needed.